src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -170,6 +170,9 @@ class Preview extends React.Component<Props> {
         this.dataDisposer = reaction(
             () => toJS(formStore.data),
             (data) => {
+                if (this.iframeRef === null) {
```

Comment:
@wachterjohannes this is necessary to prevent sending an "update" request while the iframe is not yet rendered after a locale change